tools/depends/configure.ac

```diff
@@ -382,7 +382,6 @@ case $host in
     platform_cxxflags="$platform_cflags"
     platform_os="linux"
     meson_system="linux"
-    target_platform="wayland gbm"
```

Comment:
The problem here is that we can't build for x11 on the arm/aarch64 builds. We don't cross compile the x11 libs and that's why it was left out here.

So it's still gated in the jenkins Linux build config but it's based on the target arch (like here).

In the depends/target Makefile we build dependencies based on gl/gles.

We should probably be consistent here but it's probably better to use target_platform to determine what dependencies to build.

Reply:
That's definitely a path to go as well. Either way is fine to me, so I'll let you Linux folks work out which to take

Reply:
Agreed, it makes much more sense to decide depends based on target_platform then on render system

Reply:
Ok, i'll start again. The assumption is we want to build dependencies based on target_platform.

Ive pushed a commit (and reverting the original). Absolute stab in the dark about what deps are x11/wayland, and ive no idea if theres anything that would be gbm specific. Pointers for what else you would need appreciated.